src/controllers/DrawingModeController.js

```diff
@@ -149,7 +149,7 @@ class DrawingModeController extends AnnotationModeController {
      * @return {void}
      */
     bindDOMListeners() {
-        if (this.isTouchCompatible) {
+        if (this.isMobile && this.hasTouch) {
             this.annotatedElement.addEventListener('touchstart', this.handleSelection);
```

Comment:
what if you're on a desktop browser that has touch like a surface?

Reply:
Made a ticket to clarify these checks